src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.html

```diff
@@ -111,6 +112,14 @@
           [style.visibility]="(!isCoherent(node) || hasIncoherentDependencies(node)) ? 'visible' : 'hidden'"></fa-icon>
         {{node.build.azureDevOpsBuildNumber}}
       </td>
+      <td>
+        <span [style.display]="node.build.staleness == '0' ? 'inline' : 'none'">
```

Comment:
This should probably use `*ngIf` rather than changing the display style to avoid putting extra clutter in the DOM.